File: 0627_swap_sex_of_employees/Solution.cs
// LeetCode 0627 - Swap Sex of Employees
// https://leetcode.com/problems/swap-sex-of-employees/

public class Solution {
    public const string QUERY = @"
UPDATE Salary
SET sex = CASE WHEN sex = 'm' THEN 'f' ELSE 'm' END
";
}

File: 0627_swap_sex_of_employees/Solution.kt
// LeetCode 0627 - Swap Sex Of Employees
// https://leetcode.com/problems/swap-sex-of-employees/

class Solution {
    companion object {
        const val QUERY = "UPDATE Salary\n" +
            "SET sex = CASE WHEN sex = 'm' THEN 'f' ELSE 'm' END"
    }
}
